modules/oadp-self-service-use-case-csi.adoc

```diff
@@ -0,0 +1,228 @@
+// Module included in the following assemblies:
+//
+// backup_and_restore/application_backup_and_restore/oadp-self-service/oadp-self-service-non-admin-use-cases.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-self-service-use-case-csi_{context}"]
+= Creating a non-admin backup and restore by using CSI
+
+In this use case, as a non-admin user:
+
+* You take a Container Storage Interface (CSI) backup of your application running in your authorized namespace. You do this by creating a `NonAdminBackup` (NAB) CR. The cluster administrator has enabled the `csi` plugin in the `DataProtectionApplication` (DPA). 
+* To simulate disaster, you delete the application in your authorized namespace. 
+* You then restore the application in your namespace by creating a `NonAdminRestore` (NAR) CR.
+
+.Prerequisites
+
+* You are logged in to the cluster as a non-admin user.
+* The cluster administrator has installed the {oadp-short} Operator.
+* The cluster administrator has configured the DPA to enable {oadp-short} Self-Service and the CSI plugin, as explained in section "Configuring the `DataProtectionApplication` for a non-admin CSI backup".
+* The cluster administrator has created a namespace for you and has authorized you to operate from that namespace.
+* You have installed an application in your authorized namespace. 
```

Comment:
@shubham-pampattiwar do we want to add that the non-admin user is operating or better said creating the NAB from within the namespace where the target backup application exists?